CMake/CMakeLists.txt

```diff
@@ -14,7 +11,7 @@ foreach(filename IN ITEMS
   )
   set(dest_file "${CMAKE_CURRENT_BINARY_DIR}/${filename}")
   set(expected_hash "${${filename}_expected_hash}")
-  set(url "https://raw.githubusercontent.com/Slicer/vtkAddon/${vtkAddon_version}/CMake/${filename}")
+  set(url "https://github.com/Slicer/vtkAddon/tree/main/CMake/${filename}")
```

Comment:
If a new version was pushed to the main branch of vtkAddon then the expected hashes would become all wrong. So, it is more robust if we keep using a specific hash of vtkAddon.

Could you update your PR to keep the `vtkAddon_version` variable, just set it to a more recent version?

Reply:
> Could you update your PR to keep the `vtkAddon_version` variable, just set it to a more recent version?

It has been updated.

Actually, I understand now the purpose of `raw.githubusercontent.com`, to get resource files and not html ones.